package/README.md

```diff
@@ -0,0 +1,72 @@
+# FORCE One-Step Installer Creation
+FORCE one-step installers are created using the `cx_Freeze` python package after creating a python environment using either `venv` or `conda`.
+A computer with the same operating system and architecture as the target operating system must be used to generate an installer, i.e. use a Windows machine to generate a Windows installer, a Mac (Intel) to generate a Mac installer, etc.
+Note that installers generated with Apple computers with M-series chips will not be backwards compatible with Intel-based Apple computers.
+
+Windows and macOS are the only operating systems currently supported.
+Linux users are encouraged to use pip-installed or source built versions of the RAVEN, HERON, and TEAL software packages.
+
+## 1. Build FORCE executables
+Create a conda environment `force_build_310`, install the RAVEN, HERON, and TEAL pip packages, and build the FORCE executables using the script `build_force.sh`.
```

Comment:
I think it would be better to explicitly write out the commands needed:

`conda create -n force_build_310`

`pip install ...`

Reply:
The `package/build_force.sh` script does create a conda environment force_build_310 and so forth.